import_processor/readme/USAGE.rst

```diff
@@ -0,0 +1,31 @@
+This module provides a set of pre-defined functions and wizards to facilitate importing data into Odoo.
+
+The module offers predefined variables that can be used when writing code snippets in the view.
+
+Note: Only pre-defined variables can be used in code snippets.
+
+The module consists of three main processes:
+
+1. Pre-Processor:
+   - At this stage, you can define code snippets that will be executed before importing the records in the view.
+
+2. Processor:
+   - At this stage, you can define code snippets that will be executed during the import process.
+
+3. Post-Processor:
+   - At this stage, you can define code snippets that will be executed after the records have been imported.
+
+Use Cases
+~~~~~~~~~
+
+1. Navigate to Settings > Technical > Import Processors and create a new Import Processor for CSV files with the following settings:
+   - Name: "CSV"
```

Comment:
Please don't specific any inputs here. It suggests that the naming is relevant when it's not.